src/ts/components/recommendationoverlay.ts

```diff
@@ -67,7 +67,7 @@ export class RecommendationOverlay extends Container<ContainerConfig> {
     player.on(player.exports.Event.PlaybackFinished, () => {
       // Dismiss ON_PLAYBACK_FINISHED events at the end of ads
       // TODO remove this workaround once issue #1278 is solved
```

Comment:
How do i find issue #1278?

Reply:
It doesn't exist any more. In v7, the player fired `ON_PLAYBACK_FINISHED` *before* a postroll ad would play, hence this workaround. This is not the case any longer with the new ads module in v8 so you can just remove this `if` block.